tests/app/views/test_brief_response.py

```diff
@@ -29,10 +29,27 @@ def setup(self):
                 supplier_id=0,
             )
 
-            db.session.add_all([service, brief])
+            specialist_brief = Brief(
+                data=example_listings.brief_data().example(),
+                status='live', framework_id=5, lot=Lot.query.get(6)
```

Comment:
These are set up by migrations, which is why they exist in tests. I think we shouldn't be using actual frameworks in tests at all, but it's definitely a separate discussion.